basics/concepts.md

```diff
@@ -1,79 +1,82 @@
-{% endhint %}
 
-## Pinot components
+## Physical Architecture
 
 ![](../.gitbook/assets/Pinot-Components.svg)
 
-A Pinot cluster consists of multiple distributed system components. These components are useful to understand for operators that are monitoring system usage or are debugging an issue with a cluster deployment.
+A Pinot cluster consists of the following processes, which are typically deployed on separate hardware resources in production. In development, they can fit comfortably into Docker containers on a typical laptop.
 
-* Controller
-* Broker
-* Server
-* Minion (optional)
+* **Controller**. Maintains cluster metadata and orchestrates cluster resources.
+* **Zookeeper**. Fault-tolerant, persistent storage on behalf of the controller.
+* **Broker**. Accepts queries from client processes and forwards them to servers for processing.
+* **Server**. Provides storage for segment files and compute for query processing.
+* **Minion** (optional). Distributed compute for cluster tasks other than query processing.
 
-Pinot's integration with  [Apache Zookeeper](https://zookeeper.apache.org/) and [Apache Helix](http://helix.apache.org/) allow it to be linearly scalable for an unbounded number of nodes.
+The simplest possible Pinot cluster consists of four processes: a server, a broker, a controller, and a Zookeeper node. In production environments, these are typically run on separate server instances, and scaled out as data volume, concurrent load, availability, and latency targets dictate. Production clusters exist at sizes of fewer than ten total instances to more than 1,000.
 
-{% hint style="info" %}
-Helix is a cluster management solution designed and created by the authors of Pinot at LinkedIn. Helix drives the state of a Pinot cluster from a transient state to an ideal state, acting as the fault-tolerant distributed state store that guarantees consistency. Helix is embedded as agents that operate within a controller, broker, and server, and does not exist as an independent and horizontally scaled component.
-{% endhint %}
+Pinot uses [Apache Zookeeper](https://zookeeper.apache.org/) as a distributed metadata store and and [Apache Helix](http://helix.apache.org/) for cluster management.
 
-### Pinot Controller
+Helix is a cluster management solution created by the authors of Pinot. Helix maintains a persistent, fault-tolerant map of the intended state of the Pinot cluster. It constantly monitors the cluster to ensure that the right hardware resources are allocated to implement the present configuration. When the configuration changes, it schedules or decomissions hardware resources to reflect the new configuration. When elements of the cluster change state catastrophically, it schedules hardware resources to keep the actual cluster consistent with the ideal represented in the metadata. From a physical perspective, Helix takes the form of a controller process plus agents running on servers and brokers.
```

Comment:
would it be ok to drop "catastrophically" and "actual"? also is "it" Helix or Pinot?